server/protocols/webadmin/src/main/java/org/apache/james/webadmin/WebAdminServer.java

```diff
@@ -28,17 +28,20 @@
 import org.apache.james.http.jetty.Configuration;
 import org.apache.james.http.jetty.JettyHttpServer;
 import org.apache.james.lifecycle.api.Configurable;
+import org.apache.james.webadmin.servlet.DomainServlet;
 
 import com.google.common.base.Throwables;
 
 public class WebAdminServer implements Configurable {
 
     private final JettyHttpServer server;
 
-    public WebAdminServer(Optional<Integer> port) {
+    public WebAdminServer(Optional<Integer> port, DomainServlet domainServlet) {
         this.server = JettyHttpServer.create(
             Configuration.builder()
                 .port(port)
+                .serve(Constants.DOMAIN + "/*")
```

Comment:
there's a new API for "all path below this one", @aduprat will tell you how it's named